frontend/src/libs/packages/api/base-http-api.ts

```diff
@@ -106,6 +106,15 @@ class BaseHttpApi implements HTTPApi {
       };
     }
 
+    if (response.status === HTTPCode.UNAUTHORIZED) {
+      throw new AuthError({
+        status: HTTPCode.UNAUTHORIZED,
+        errorType: ServerErrorType.AUTHORIZATION,
+        message: parsedException.message,
+        details: 'details' in parsedException ? parsedException.details : [],
```

Comment:
Not sure whether we need errorType, details and status

Reply:
Deleted those!